tests/evaluate/test_evaluation.py

```diff
@@ -1035,6 +1035,46 @@ def get_evaluate_call_arg(model, evaluator_config):
                     )
 
 
+def test_custom_evaluators_no_model_or_preds(multiclass_logistic_regressor_model_uri, iris_dataset):
```

Comment:
Verify that custom evaluators can are called correctly when neither the model nor predictions are specified